### DISCUSSION TITLE
Where and how is the access token stored in the mobile version of the plugin?

### DISCUSSION THREAD

**Discussion by aaueer** [Vinzent03/obsidian-git#651] [Q&A] [answered]
In the desktop version, the authentication is handled by the git installation itself or rather the solution it is setup to use (as far as I know). The mobile version uses isomorphic-git. When passing the token to the plug-in, where and how does it get stored? Is it handled by isomorphic-git or the Plugin? I could not find a conclusive answer, but to be fair I tend to be really blind ^^"

**Comment by Vinzent03** [comment, marked as answer]
It's managed by the plugin and I'm using `window.localstorage` (LocalStorage)

**Comment by aaueer** [comment]
Sorry for the late reply and thanks for the answer! I was just wondering whether it is stored in plain text somewhere. Also Happy New Year!
